classes/sysvinit.oeclass

```diff
@@ -7,7 +7,11 @@
 ## <chosen name> (declare this to RECIPE_FLAGS)
 ## SYSVINIT_SCRIPT_<chosen name> Set this to the name of your initscript.
 ## <chosen name>_sysvinit_start Set this to the start level.
-## <chosen name>_sysvinit_stop (optional) Set this to the stop level.
+## <chosen name>_sysvinit_stop Set this to the stop level.
+##
+## To omit the start/stop script, set <chosen name>_sysvinit_start /
+## <chosen name>_sysvinit_stop to anything but a two-digit string, the
+## string "disable" being the preferred way.
```

Comment:
I think False would convey the same intention, and that would allow us to replace a lot of existing instances, and still be backwards-compatible with code that uses "0", and forward-compatible with a potential semantic change of the False/True tokens.